neptune/new/run.py

```diff
@@ -69,6 +71,19 @@ def inner_fun(self, *args, **kwargs):
     return inner_fun
 
 
+LEGACY_METHODS = (
+    'create_experiment',
```

Comment:
`create_experiment`  isn't part of `Run` class.

It should be defined in `neptune.new` similar to other old `neptune` methods like:
* `set_project`
* `get_experiment`
* `append_tag`
* etc.

Reply:
1. you're right, I'll add old `neptune` methods to `neptune.new`
2. I'd argue that this one actually makes sense, too. I think this is a legit flow for old API:
```
project = neptune.init(...)
project.create_experiment(...)
```

Reply:
Ad 2.
Yes